### PR TITLE
[MM-40750] Remove 'Status' Menu Header on Status Dropdown

### PR DESCRIPTION
**Pull request by mylonsuren**
#### Summary

This PR fixes an issue where a menu header with text 'Status' appeared when custom status was disabled. The menu header would not appear if custom status was enabled (as expected).

#### Ticket Link

Fixes [MM-40750](https://mattermost.atlassian.net/browse/MM-40750)

#### Screenshots

|  Before  |  After  |
|----|----|
| MM-40750 Before |MM-40750 After |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/status_dropdown/status_dropdown.tsx [hunk 1]
```diff
@@ -391,14 +391,6 @@ export default class StatusDropdown extends React.PureComponent<Props, State> {
                     ariaLabel={localizeMessage('status_dropdown.menuAriaLabel', 'Set a status')}
                     id={'statusDropdownMenu'}
                 >
```
[the remark this answers is not in the document]
Comment (mylonsuren) [reply components/status_dropdown/status_dropdown.tsx:393]: No nothing breaks as this component wasn't appearing anyway when custom status was enabled.

[the remark this answers is not in the document]
Comment (mkraft) [reply components/status_dropdown/status_dropdown.tsx:393]: > No nothing breaks as this component wasn't appearing anyway when custom status was enabled.

Perfect. I only saw code being removed so just wanted to confirm. Thanks.

**Review by mkraft — APPROVED** [review]

**Review by BenCookie95 — APPROVED** [review]

**Review by furqanmlk — APPROVED** [review]
Working as expected

**Before**
image


**After**

image

**Pull request merged**